tests/integration/components/card-test.js

```diff
@@ -0,0 +1,92 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, click } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | card', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders the card component', async function (assert) {
+    await render(hbs`<Card />`);
+
+    assert.dom('.card').exists();
+    assert.dom('.card').hasNoText();
+  });
+
+  test('it renders information about a profile service and validates is buttonOnClick function called', async function (assert) {
+    let callCount = 0;
+    this.setProperties({
+      title: 'Qualification Criteria',
+      description:
+        'To update your profile details, link your profile service URL with RealDevSquad Service.',
+      buttonText: 'Get Started',
+      onButtonClickEventHandler: () => {
+        ++callCount;
+      },
+    });
+
+    await render(
+      hbs`<Card 
+      @title={{this.title}} 
+      @description={{this.description}}
+      @buttonText={{this.buttonText}}
+      @onButtonClickEventHandler={{this.onButtonClickEventHandler}}
+      />`
+    );
+
+    assert.dom('.card').exists();
+    assert.dom('.card .card__title').exists();
+    assert.dom('.card .card__title').hasText(this.title);
+    assert.dom('.card .card__description').exists();
+    assert.dom('.card .card__description').includesText(this.description);
+    assert.dom('.card .card__actions').exists();
+    assert.dom('.card .card__actions .card__button').exists();
+    assert.dom('.card .card__actions').includesText(this.buttonText);
```

Comment:
We can use `data-test-selectors` instead of classNames here which the Ember testing setup provides.
Read more from here :
https://guides.emberjs.com/release/testing/testing-tools/#toc_ember-test-selectors

Reply:
I have re-written tests using `data-test-selectors`. 
Check this comment: https://github.com/Real-Dev-Squad/website-my/pull/355/commits/d1935c3e247915fc6834a6dbd82318dc440c449d